marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -228,12 +228,14 @@ public ResponseEntity<Void> clearDraftModels() {
     }
 
     private void deleteDraftsInDB(DatabaseClient client) {
+        //clearing the draft in respective databases
         QueryManager qm = client.newQueryManager();
-        //publish models to ES collection
         DeleteQueryDefinition def=qm.newDeleteDefinition();
         def.setCollections("http://marklogic.com/entity-services/models/draft");
-        //clearing the draft in respective databases
         qm.delete(def);
+        DeleteQueryDefinition defConcept =qm.newDeleteDefinition();
```

Comment:
Why not set the collections once (since the setCollections can handle an array) and only call the delete one time?

Reply:
There is a limitation in client-api so we are forced to do two separate calls.